### PR TITLE
added petcu2022replication

### PR DESCRIPTION
**Pull request by jeroenwijnen98**
## Project Name
petcu2022replication

## Compiled PDF
[Compiled Pdf](https://github.com/jeroenwijnen98/MLRC/blob/main/2021/petcu2022replication/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors
No more errors

### CODE REVIEW

File: 2021/petcu2022replication/journal/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+  - name: Pim Praat
+    orcid: 0000-0003-1412-3389
+    email: pimpraat@gmail.com
+    affiliations: 1      # * is for contact author
+
+  - name: Jeroen Wijnen
+    orcid: 0000-0002-1145-3596
+    email: j.wijnen@outlook.com
+    affiliations: 1*
```
Comment (koustuvsinha) [inline 2021/petcu2022replication/journal/metadata.yaml:27]: Please add a comma before `*`

File: 2021/petcu2022replication/journal/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+keywords:  rescience c, machine learning, online selection, prophet problem, secretary problem, fairness, bias, Python
+
+# Code URL and DOI/SWH (url is mandatory for replication, doi after acceptance)
+# You can get a DOI for your code from Zenodo, or an SWH identifier from
+# Software Heritage.
+#   see https://guides.github.com/activities/citable-code/
+code:
+  - url: https://github.com/pimpraat/FACT-Ai
+  - doi: 00.0000/zenodo.0000000
+  - swh: swh:1:dir:aaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```
Comment (koustuvsinha) [inline 2021/petcu2022replication/journal/metadata.yaml:52]: Please provide a proper SWH id

**Comment by jeroenwijnen98** [comment]
- On the first page of pdf reviewer data is covering the text.
- Formatting with double column, as on page 3 see [Compiled Pdf](https://github.com/jeroenwijnen98/MLRC/blob/main/2021/petcu2022replication/journal/article.pdf)
- It doesn't understand in-text reference to figures / tables etc
- When building, it gives an error because it cannot find citations
		Error message can be found [here](https://pastebin.com/0bS7vmYy)

**Comment by jeroenwijnen98** [comment]
After extensive rewriting of the Latex code, the issues were resolved.
Main problems were:
	- Latex errors which were not problematic in Overleaf, but were in the local compilation
	- incorrect reference to bibliography.bib in metadata.yaml

**Comment by GuillyK** [comment]
Hey, what was the problem with "It doesn't understand in-text reference to figures / tables etc"? Because we have the same problem. Our bibliography is correctly mentioned in the metadata.yaml file.

**Comment by jeroenwijnen98** [comment]
Did the last fixes. Mainly to do with making sure our algorithm pseudocode runs properly

**Comment by jeroenwijnen98** [comment]
> Hey, what was the problem with "It doesn't understand in-text reference to figures / tables etc"? Because we have the same problem. Our bibliography is correctly mentioned in the metadata.yaml file.

Hey, our problem was twofold:
- First, we had to copy our folder with used images to the journal file. For some reason it also needs to be there.
- Second, our code contained several faulty Latex lines, which didn't cause problems in our Overleaf file, but somehow did when compiling locally. The most annoying thing is that it didn't properly give us an error message for those. It just gave an error that citations could not be found & created overlap on first page with editor information. We solved this by iteratively adding Latex lines to content.tex and compiling. And as soon as the code broke again, debugging the new lines. this was a very cumbersome process and took multiple hours...
Hope this helps!

**Comment by GuillyK** [comment]
Thanks for the help! Ours has the same problem with no errors in overleaf but errors in the compiling. Your solution is probably the most reliable to solve the errors. We will do that then too.

**Pull request merged**